tests/pytests/test_json.py

```diff
@@ -999,3 +999,32 @@ def testVector_delete(env):
     env.assertEqual(conn.execute_command('JSON.DEL', 'j5'), 1)
 
     env.expect(*q).equal([1, 'j6'])
+
+
+@no_msan
+def testRedisCommands(env):
+    env.execute_command('FT.CREATE', 'idx', 'ON', 'JSON', 'SCHEMA', '$.t', 'TEXT', '$.flt', 'NUMERIC')
+    env.execute_command('JSON.SET', 'doc:1', '$', r'{"t":"riceratops","n":"9072","flt":97.2}')
+    env.expect('ft.search', 'idx', 'ri*', 'NOCONTENT').equal([1, 'doc:1'])
+
+    # Test Redis COPY
+    env.execute_command('COPY', 'doc:1', 'doc:2')
+    env.execute_command('COPY', 'doc:2', 'dos:3')
+    env.expect('ft.search', 'idx', 'ri*', 'NOCONTENT').equal([2, 'doc:1', 'doc:2'])
```

Comment:
Shouldn't `dos:3` also be in search result?

Reply:
My mistake the schema should have doc: as prefix